src/site/content/en/blog/min-max-clamp/index.md

```diff
@@ -0,0 +1,179 @@
+  <figcaption class="w-figcaption">
+    Using the clamp() function to limit a minimum and maximum width.
+  </figcaption>
+</figure>
+
+By the same token, you can ensure a minimum size for legible text using the `max()` function. This would look like: `width: max(45ch, 50%);`. Here, the browser selects whichever is larger, `45ch` or `50%`, meaning the element must be at *least* `45ch` or larger.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/min-max-clamp/min-width.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Using the clamp() function to limit a minimum and maximum width.
+  </figcaption>
+</figure>
+
+## Padding Management
+
+Using the same concept as above, where the `min()` function can set a “max” value and `max()` sets a “min” value, you can use `max()` to set a minimum padding size. This example comes from [CSS Tricks](https://css-tricks.com/using-max-for-an-inner-element-max-width/), where reader Caluã de Lacerda Pataca shared this idea:
+The idea is to enable an element to have additional padding at larger screen sizes, but maintain a minimum padding at smaller screen sizes, particularly on the inline padding. To achieve this, you can use `calc()` and subtract the minimum padding from either side: `calc((100vw - var(--contentWidth)) / 2)`, *or* you can use max: `max(2rem, 50vw - var(--contentWidth) / 2)`. All together it looks like:
+
+```css
+footer {
+  padding: var(--blockPadding) max(2rem, 50vw - var(--contentWidth) / 2);
+}
+```
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/min-max-clamp/min-padding.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Setting a minimum padding for a component using the max() function. <a href="https://codepen.io/chriscoyier/pen/qBZqNKa">See Demo on Codepen.</a>
+  </figcaption>
+</figure>
+
+## Fluid Typography
+
+In order to enable [fluid typography](https://www.smashingmagazine.com/2016/05/fluid-typography/), [Mike Riethmeuller](https://twitter.com/mikeriethmuller) popularized a technique that uses the `calc()` function to set a minimum font size, maximum font size, and allow for scaling from the min to the max.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/min-max-clamp/fliud-type.mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    Creating fluid typography with clamp(). <a href="https://codepen.io/una/pen/ExyYXaN">See Demo on Codepen.</a>
+  </figcaption>
+</figure>
+
+Utilizing `clamp()`, you can now write this more clearly. Rather than requiring a complex string, you can let the browser do the work for you. You can set the minimum acceptable font size (for example, `1.5rem` for a title, maximum size (i.e. `3rem`) and ideal size of `5vw`. 
```

Comment:
```suggestion
With `clamp()`, you can write this more clearly. Rather than requiring a complex string, the browser can do the work for you. Set the minimum acceptable font size (for example, `1.5rem` for a title, maximum size (i.e. `3rem`) and ideal size of `5vw`. 
```